roles/lnls-ans-role-java/vars/Debian-buster.yml

```diff
@@ -1,8 +1,9 @@
 ---
-# JDK version options include:
-#   - java
-#   - openjdk-8-jdk
-__java_packages:
-  - openjdk-11-jdk
-  - openjdk-11-jre
-  - openjfx
+  # JDK version options include:
+  #   - java
+  #   - openjdk-8-jdk
+  __java_packages:
+    - openjdk-8-jdk
+    - openjdk-8-jre
+    - openjfx
+  
```

Comment:
my github viewer complain about the end of this file. It says: "no newline at end of file".

Reply:
please check if it stopped complaining...

Reply:
sorry @xresende, I only saw this now! I checked and it is ok! There are other issues like this though.